From: Patrick Griffis <tingping@tingping.se>
Date: Thu, 23 Aug 2018 17:28:26 -0400
Subject: Add /proc to blacklist

This doesn't make sense to ever be in the sandbox

Closes: #2008
Approved by: alexlarsson

(cherry picked from commit 738c11324a4f0d0117e0f65889ae2d26106fde94)
---
 common/flatpak-exports.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/flatpak-exports.c b/common/flatpak-exports.c
index 6f4411d..92410cb 100644
--- a/common/flatpak-exports.c
+++ b/common/flatpak-exports.c
@@ -51,7 +51,7 @@
 /* We don't want to export paths pointing into these, because they are readonly
    (so we can't create mountpoints there) and don't match whats on the host anyway */
 const char *dont_export_in[] = {
-  "/lib", "/lib32", "/lib64", "/bin", "/sbin", "/usr", "/etc", "/app", "/dev", NULL
+  "/lib", "/lib32", "/lib64", "/bin", "/sbin", "/usr", "/etc", "/app", "/dev", "/proc", NULL
 };
 
 static char *
